### ISSUE TITLE
how do you do [[filenames with spaces]]?

### ISSUE THREAD

**Issue by cedardocster** [svsool/vscode-memo#195]
Most markdown extensions force you to use [[dashes-in-filenames]] but from your help page

https://github.com/svsool/vscode-memo/tree/master/help

it appears that yours does not.

However, when I install your extension (plus the Markdown Preview Enhanced extension), I still get the error when I click on the [[name of some file]] link in the preview pane that the file does not exist and its looking for [[name_of_some_file]] instead of the one with the spaces.

Can you explain how to do this for non-programmers (i.e. "for dummys", I'm just looking for a personal wiki note solution and this seems to fit the bill if I can just get it working right)

Thank you,
- Dave

**Comment by svsool** [comment]
Memo does not do anything specific about filenames. Likely you have some other extensions installed that interfere with Memo. Try to disable all extensions and leave out only Memo enabled, does it work after? All links in the help folder should be followable on click and work away. I tried Memo with Markdown Preview Enhanced and it works just fine.

**Comment by cedardocster** [comment]
Markdown Preview was the only other extension I had.  But I disabled it and I get the instruction to ctrl-click the link on the editor side to create it, but it doesn't actually do anything.  I'm on Ubuntu 20.04 btw.
![editorSide](https://user-images.githubusercontent.com/10538705/99336484-9c484f00-283e-11eb-97f9-a697cc63557b.png)

**Comment by svsool** [comment]
Hello Dave, could you please download [help](https://github.com/svsool/vscode-memo/tree/master/help) folder, open it as a workspace and check if links work for you?

**Comment by cedardocster** [comment]
Thank you for helping me try to figure this out.

I downloaded the folder into my /tmp/ folder with this
svn checkout https://github.com/svsool/vscode-memo/trunk/help

When I did "file > open workspace" there was nothing to open.  So I did file > open folder and was able to view the README.md file and when I added [[some file]] I *was able to* ctrl-click the link on the edit side to make the new file with the space instead of the underscores.

But when I then close Vscode and open it again and open my notes folder, the same process still does not work.  Could the size of the folder make a difference?  I have over 5000 md files in there.

I'll post this and then try making a blank folder with a file, try the same process, and if successful will try pasting the rest of the files in there to see if that's what makes a difference.

Thanks again for the help - I would love to get this working!! :D 
- Dave

**Comment by cedardocster** [comment]
Okay, tried putting just a single md file in a new empty folder, opened it with VScode and it still didn't work.  Then I tried your README.md file in there and still no new link creation ability.  Is there some magical file in one of the subfolders that allows this to happen?

**Comment by cedardocster** [comment]
I figured it out - I was going to the folder and right clicking the file and clicking "open with" > "VSCode".   I finally noticed a message on the left pane "you have not yet opened a folder".  I had never opened whole folders before, just the individual files, and somehow any [[singleWordLinks]] would work when you click on the links, so I never knew operating that way was a problem.

So, anyway - Thanks so much for the push in the right direction.  Now I know to open folders, not just files :)
